websites/D/Dumpert/presence.ts

```diff
@@ -0,0 +1,94 @@
+const presence = new Presence({
+  clientId: "840126038205923369"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
+let title: Element, title2: Element;
+
+presence.on("UpdateData", async () => {
+const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  },
+  page = window.location.pathname,
+  pageh = document.location.href;
+presenceData.startTimestamp = browsingStamp;
+
+function waitForElement(){
+  if(title2 != undefined){
+    if (title2.className.includes("paused")) {
+      delete presenceData.endTimestamp;
+      presenceData.smallImageKey = "pause";
+    } else if (title2.className.includes("playing")) {
+      const currentTime = presence.timestampFromFormat(
+        document.querySelector(
+          `#vjs_video_${title2.className.slice(40, 55).replace(/[^0-9.]/g, "")} > div.vjs-control-bar.progress-in-menu > div.vjs-current-time.vjs-time-control.vjs-control > span.vjs-current-time-display`
+        ).textContent
+      ),
+      
+      durationss = presence.timestampFromFormat(
+        document.querySelector(
+          `#vjs_video_${title2.className.slice(40, 55).replace(/[^0-9.]/g, "")} > div.vjs-control-bar.progress-in-menu > div.vjs-duration.vjs-time-control.vjs-control > span.vjs-duration-display`
+        ).textContent
+      ),
+      timestamps = presence.getTimestamps(currentTime, durationss);
+      presenceData.endTimestamp = timestamps[1];
+      presenceData.smallImageKey = "play";
+    }
+  }
+  else{
+      setTimeout(waitForElement, 3000);
+  }
+}
+
+if (page == "/") {
+  presenceData.details = "Bekijkt:";
+  presenceData.state = "De Home Pagina";
+} else if (pageh.includes("plaatjes")) {
+  let element = document.querySelector('meta[property~="og:title"]');
+  if (!element) {
+    presenceData.details = "Bekijkt:";
+    presenceData.state = "Plaatjes";
+  } else {
+let content = element && element.getAttribute("content");
+ presenceData.details = content;
+  }
+} else if (pageh.includes("filmpjes")) {
+  let element = document.querySelector('meta[property~="og:title"]');
+  if (!element) {
+    presenceData.details = "Bekijkt:";
+    presenceData.state = "Filmpjes";
+  } else {
+  let content = element && element.getAttribute("content");
```

Comment:
let because of what? where you re-assign it?
```suggestion
const content = element && element.getAttribute("content");
```